samples/sensor_monitoring/src/sm_buttons.c

```diff
@@ -164,12 +166,14 @@ sid_time_t sm_buttons_press_time_get(uint8_t btn_id)
 
 int sm_buttons_init()
 {
-	button_set_action_long_press(DK_BTN1, btn_event_handler, BTN_EVENT_RESET_KEY);
+	button_set_action_long_press(DK_BTN1, btn_event_handler, BTN_EVENT_KEY_0);
 	button_set_action_short_press(DK_BTN1, btn_event_handler, BTN_EVENT_KEY_0);
+#ifndef CONFIG_BOARD_THINGY53_NRF5340_CPUAPP
+	button_set_action_long_press(DK_BTN1, btn_event_handler, BTN_EVENT_RESET_KEY);
```

Comment:
I don't get it. For Thingy we set BTN_EVENT_KEY_0 for long and short press?
Shouldn't we use Button 1 long press for reset, and short presses for events?

e.g.
```
	button_set_action_long_press(DK_BTN1, btn_event_handler, BTN_EVENT_RESET_KEY);
	button_set_action_short_press(DK_BTN1, btn_event_handler, BTN_EVENT_KEY_0);
#ifndef CONFIG_BOARD_THINGY53_NRF5340_CPUAPP
	button_set_action(DK_BTN2, btn_event_handler, BTN_EVENT_KEY_1);
	button_set_action(DK_BTN3, btn_event_handler, BTN_EVENT_KEY_2);
	button_set_action(DK_BTN4, btn_event_handler, BTN_EVENT_KEY_3);
#endif
```

Reply:
This was changed for the DEMO to prevent accidental factory reset, I will roll back the change as suggested